packages/yew/src/html/conversion/into_prop_value.rs

```diff
@@ -182,10 +183,13 @@ macro_rules! impl_into_prop {
 }
 
 // implemented with literals in mind
-impl_into_prop!(|value: &'static str| -> String { value.to_owned() });
+impl_into_prop!(<'a> |value: &'a str| -> String { value.to_owned() });
 impl_into_prop!(|value: &'static str| -> AttrValue { AttrValue::Static(value) });
 impl_into_prop!(|value: String| -> AttrValue { AttrValue::Rc(Rc::from(value)) });
 impl_into_prop!(|value: Rc<str>| -> AttrValue { AttrValue::Rc(value) });
+impl_into_prop!(|value: Cow<'static, str>| -> AttrValue { AttrValue::from(value) });
+impl_into_prop!(<'a> |value: Cow<'a, str>| -> String { Cow::into_owned(value) });
```

Comment:
What if Cow is Borrowed? I don't think we should be implicitly cloning it here.

Reply:
we already implicitly convert a `&str` into `String`, implicitly converting a `Cow<'_, str>`, which is either a `String` or a `&'a str`, to `String` is just a logical continuation in my opinion